### PR TITLE
Java SDK - Log4j Hotfix

### PR DESCRIPTION
**Pull request by DerAusHeidelberg**
- Updated to newer Version (2.15.0) of Log4j to fix the Zero-Day-Exploit

### CODE REVIEW

**Review by Ryouzanpaku — APPROVED** [review]
approven.

File: src/test/java/com/unzer/payment/business/paymenttypes/PaypalTest.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public void testCreatePaypalWithEmail() throws HttpCommunicationException {
 		paypal.setEmail("test.user@email.com");
 		paypal = (Paypal) getUnzer().createPaymentType(paypal);
 		assertNotNull(paypal.getId());
-		assertEquals(null, paypal.getEmail());
+		assertEquals("test.user@email.com", paypal.getEmail());
```
Comment (sixer1182) [inline src/test/java/com/unzer/payment/business/paymenttypes/PaypalTest.java:53]: why did this change?

File: src/test/java/com/unzer/payment/communication/JsonDateConverterTest.java [hunk 2]
```diff
@@ -14,26 +14,24 @@
...
     public void getDateTestFormatDateTime(){
-        System.setProperty("user.timezone", "UTC");
         JsonObject jsonObject = new JsonObject();
         jsonObject.addProperty("date", "1980-12-01 12:00:00");
 
 
         JsonDateConverter jsonDateConverter = new JsonDateConverter();
         Date jsonDate = jsonDateConverter.getDate(jsonObject.get("date"));
-        assertEquals("Mon Dec 01 12:00:00 UTC 1980", jsonDate.toString());
+        assertEquals("Mon Dec 01 13:00:00 CET 1980", jsonDate.toString());
```
Comment (sixer1182) [inline src/test/java/com/unzer/payment/communication/JsonDateConverterTest.java:23]: why did this change?

File: src/test/java/com/unzer/payment/communication/JsonDateConverterTest.java [hunk 3]
```diff
@@ -14,26 +14,24 @@
...
     public void getDateTestFormatDate(){
-        System.setProperty("user.timezone", "UTC");
         JsonObject jsonObject = new JsonObject();
         jsonObject.addProperty("date", "1980-12-01");
 
 
         JsonDateConverter jsonDateConverter = new JsonDateConverter();
         Date jsonDate = jsonDateConverter.getDate(jsonObject.get("date"));
-        assertEquals("Mon Dec 01 00:00:00 UTC 1980", jsonDate.toString());
+        assertEquals("Mon Dec 01 01:00:00 CET 1980", jsonDate.toString());
```
Comment (sixer1182) [inline src/test/java/com/unzer/payment/communication/JsonDateConverterTest.java:34]: why did this change?

**Review by sixer1182 — COMMENTED** [review]
Some questions about the update.

**Pull request merged**